.prettierignore

```diff
@@ -1,4 +1,18 @@
 .cache/
+.editorconfig
+.eslintignore
+.gitignore
+.prettierignore
+CODEOWNERS
+LICENSE
+coverage
+core
 dist/
-
-*.html
\ No newline at end of file
+node_modules
+yarn.lock
```

Comment:
I think you should include the API spec file too. Generated files should never be "prettified" IMO

Reply:
Yes!